### PR TITLE
:fire: Failing push notification integration tests

### PR DESCRIPTION
**Pull request by Odrakir**
## What's this?
This removes some failing tests while we decide what to do with them. They are testing stuff that's not on production yet, so we are safe for now.

The tests were failing because we were using emails filtered out by backend (they only allow `@toggl.com` emails)

Supersedes #5566 

## How is it done?
I'm deleting all the tests, there were some on the base class that were passing, but I have my doubts to the validity of those tests, as the emails were wrong.

## :squid: Permissions
Go!

### CODE REVIEW

File: Toggl.Networking.Tests.Integration/Toggl.Networking.Tests.Integration.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (juanlaube) [inline Toggl.Networking.Tests.Integration/Toggl.Networking.Tests.Integration.csproj]: This seems to be a BOM character. Please undo the changes on this file 🙂

**Comment by amulware** [comment]
I want this merged.

**Comment by amulware** [comment]
Integration tests (currently in progress) from this branch: https://app.bitrise.io/build/7f49bbf1dbd9d962

**Comment by amulware** [comment]
Nevermind the test run above. The deleted tests are obviously no longer failing, but we have tons of other failing tests right now, so hard to tell what's going on really.

**Comment by amulware** [comment]
Tests passed eventually here: https://app.bitrise.io/build/c2859c90cae7365d
